Assets/_Scripts/FieldState.cs

```diff
@@ -3,101 +3,34 @@
 using UnityEngine;
 using UnityEngine.UI;
 using System.Linq;
-using UnityEngine.SceneManagement;
 
+// Class that holds field information / helper functions
 public class FieldState : MonoBehaviour
```

Comment:
It was originally this way, but I felt (And I think you did too) that TurnController was getting too cluttered with too many operations that don't deal with the core game loop, which should be the main thing that it does. 

I do agree that maybe "FieldState" is a misleading class name for these functions. Maybe instead it should be renamed "BattleField" to align with Battle and BattleFight? I'm not in favour of moving these functions into TurnController though.

Reply:
I was thinking, we could break TurnController into multiple classes, a class for each phase of the turn. That was we can solve this issue.

Reply:
Sounds like it could be a good idea, but it may take some effort. I think TurnController is good enough for now, but if you want to work on this, feel free to.

Reply:
We can keep it for now. Just wanted raise awareness.